cobra/mit/request.py

```diff
@@ -1932,11 +2004,27 @@ def mode(self, value):
             * createsession
             * interactive
             * generatereport
+            * getsessiondetail
+            * getsessionslist
+            * getreportstatus
+            * getreportslist
+            * atomiccounter
+            * deletesession
+            * modifysession
+            * clearreports
+            * traceroute
+            * span
 
         Raises:
           ValueError: If the value is not a valid value.
         """
-        allowedValues = {'createsession', 'interactive', 'generatereport'}
+        allowedValues = {
```

Comment:
The signature @mtimm length-sorted orthogonal dict/list definition. So beautiful. One day I will learn how to do this